### ISSUE TITLE
[phys-services] Delete I3TRandom (Trac #2137)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2129]
nobody uses it

Migrated from https://code.icecube.wisc.edu/ticket/2137
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:14:43", 
    "description": "nobody uses it", 
    "reporter": "kjmeagher", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067283824229", 
    "component": "combo core", 
    "summary": "[phys-services] Delete I3TRandom", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2018-02-14T22:13:51", 
    "milestone": "", 
    "owner": "kjmeagher", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20180220T14:27:21: 
* kjmeagher changed resolution from "" to "fixed"
* kjmeagher commented:

> In changeset 161127:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="161127"
> remove unused random service `I3TRandomService`, closes #2129
> ```


* kjmeagher changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:14:43: 
* olivas commented:

> Milestone IceRec 5.3 deleted


* olivas changed milestone from "IceRec 5.3" to ""
